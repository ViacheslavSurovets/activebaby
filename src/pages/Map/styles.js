import styled from 'styled-components';
import { desktop } from '@utils/media';

export const WrapperMap = styled.div`
 display: flex;
 justify-content: center;
 align-items: center;
 height: 100%;
 ${desktop}{
  height: 50vw;
 }
 width: 100%;
`;
