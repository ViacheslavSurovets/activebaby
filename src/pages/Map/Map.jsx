import React from 'react';
import { GoogleMaps } from '@components/GoogleMaps';
import { WrapperMap } from './styles';

const Map = () => (
  <WrapperMap>
    <GoogleMaps />
  </WrapperMap>
);

export default Map;
