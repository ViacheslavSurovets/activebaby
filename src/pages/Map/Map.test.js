import React from 'react';
import { shallow } from 'enzyme';
import Map from './Map';


describe ( 'Map', () => {

  it ( 'Auth Snapshot', () => {
    expect ( shallow ( <Map  /> ) ).toMatchSnapshot ();
  } );
} );
