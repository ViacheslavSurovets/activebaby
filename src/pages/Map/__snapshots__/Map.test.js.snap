// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map Auth Snapshot 1`] = `ShallowWrapper {}`;
